gpm_core/src/lib.rs

```diff
@@ -1 +1,7 @@
 pub mod package;
+pub mod package_writer;
+pub mod store_project;
+
+pub const TOML_CONFIG_PATH: &str = "config.toml";
+pub const JSON_CONFIG_PATH: &str = "config.json";
+pub const IGNORE_PATH: &str = ".modignore";
```

Comment:
Perhaps the constants could be placed in the `constants.ws` file/mod to avoid polluting the global namespace?

Reply:
As of now, we don't have too many constant to move them in another file. Will however place them in a module.